src/ux/step1/WalletProviders.tsx

```diff
@@ -22,6 +25,21 @@ const ProvidersWrapper = styled.div`
   padding: 8px;
 `
 
+const LanguageSelector = styled.select`
+  float: left;
+  border: 0;
+`
```

Comment:
It is missing the color, size, and removal of the outline when it is focused. Try this:

```suggestion
const LanguageSelector = styled.select`
  float: left;
  border: 0;
  color: #B0AEB1;
  size: 12px;
  &:focus {
    outline: none;
  }
`
```